src/codec_rav1e.c

```diff
@@ -208,9 +208,9 @@ static avifResult rav1eCodecEncodeImage(avifCodec * codec,
         // It may also save 3 bytes since the AV1 encoder may set color_description_present_flag to 0
         // (see Section 5.5.2 "Color config syntax" of the AV1 specification).
         rav1e_config_set_color_description(rav1eConfig,
-                                           RA_MATRIX_COEFFICIENTS_UNSPECIFIED,
-                                           RA_COLOR_PRIMARIES_UNSPECIFIED,
-                                           RA_TRANSFER_CHARACTERISTICS_UNSPECIFIED);
+                                           (RaMatrixCoefficients)image->matrixCoefficients,
+                                           (RaColorPrimaries)image->colorPrimaries,
+                                           (RaTransferCharacteristics)image->transferCharacteristics);
```

Comment:
The comment above is no longer accurate. I suggest replacing it with a link to the issue and/or a short explanation:

> CICP values could be set to 2/2/2 (Unspecified) in the Sequence Header OBU for simplicity and to save 3 bytes, but some decoders ignore the `colr` box and rely on the OBU contents instead. See https://github.com/AOMediaCodec/libavif/issues/2850.

To be done in other AV1 codec wrappers too.